torch/_dynamo/eval_frame.py

```diff
@@ -519,8 +523,15 @@ def guard_export_print(guards):
 
 
 def export(
-    f, *args, aten_graph=False, decomposition_table=None, tracing_mode="real", **kwargs
-):
+    f: Callable[[Any], Any],
```

Comment:
This makes it unnecessarily verbose imo. `Callable` alone fails with
```
  Error (MYPYNOFOLLOW) [type-arg]
    Missing type parameters for generic type "Callable"
```
I wonder if this rule should be relaxed in general.